GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1374,7 +1554,7 @@ private string GetNewProjectedShaForPlaceholder(string path)
             PlaceholderListDatabase.PlaceholderData placeholder,

```

Comment:
Not for this PR but the method name `ProcessGvUpdateDeletePlaceholderResult` needs to change at some point.